### PR TITLE
Revisions to control plane topics.

### PR DESCRIPTION
**Pull request by JStickler**
This PR
    • Addresses  https://issues.jboss.org/browse/OSSM-42, https://issues.jboss.org/browse/OSSM-43, and https://bugzilla.redhat.com/show_bug.cgi?id=1752159. 
    • Standardizes headings
    • Adds some context around the tasks
    • Splits the member roll topic into "creating" and "modifying" topics

Procedures have been verified against OCP 4.1.

@brianredbeard, @brian-avery, @tvieira, @ yxun

### CODE REVIEW

File: modules/ossm-member-roll-create.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+.Procedure
+
+. Log in to the {product-title} CLI.
++
+----
+$ oc login
+----
++
+. Create `ServiceMeshMemberRoll` resource in the same project as the `ServiceMeshControlPlane` resource, in our example that is `istio-system`. The resource must be named `default`.
```
Comment (brian-avery) [inline modules/ossm-member-roll-create.adoc]: Create a `ServiceMeshMemberRoll`

File: modules/ossm-member-roll-modify.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Installed Operators*.
+
+. Click the *Project* menu and choose the project where your `ServiceMeshControlPlane` is deployed from the list, for example `istio-system`.
+
+. Click the {ProductName} Operator.
```
Comment (brian-avery) [inline modules/ossm-member-roll-modify.adoc]: Should all of these `click` items be enclosed in asterisks? Or is there a precedence here?

From: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc

Use single asterisks for web console / GUI items (menus, buttons, page titles, etc.). Use two characters to form the arrow in a series of menu items, ->

**Review by brian-avery — APPROVED** [review]

**Comment by pbajjuri20** [comment]
JStickler: Kiali QE tested and passed

File: modules/ossm-member-roll-modify.adoc (hunk 2 above)
Comment (JStickler) [reply modules/ossm-member-roll-modify.adoc]: The formatting is to call out the button name or the menu name.  You don't have to apply formatting to the action (Click, Select).  See the examples down in the Quick Markup Reference section.

**Review by bergerhoffer — COMMENTED** [review]
Just a few things.

File: modules/ossm-control-plane-deploy.adoc [hunk 3]
```diff
@@ -5,44 +5,57 @@
...
-== Deploying the control plane with the web console
+== Deploying the control plane from the web console
 
 Follow this procedure to deploy the {ProductName} control plane by using the web console.
 
 .Prerequisites
 
 * The {ProductName} Operator must be installed.
-* Review the xref:../service_mesh_install/customizing-installation-ossm.adoc#customize-installation-ossm[Customize the {ProductName} installation] instructions.
+* Review the instructions for how to xref:../service_mesh_install/customizing-installation-ossm.adoc#customize-installation-ossm[Customize the {ProductName} installation].
```
[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-control-plane-deploy.adoc]: Bah, I forgot that rule when I was breaking up the assembly!  * deletes the x-ref *

File: modules/ossm-member-roll-create.adoc [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
++
+[NOTE]
+====
+It can take a short time for the Operator to finish copying the resources, therefore you may need to refresh the screen to see the *Create Istio Service Mesh Member Roll* button.
+====
+
+. On the Create Service Mesh Member Roll page, modify the YAML to add your projects as members.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+. Click *Create* to save the Service Mesh Member Roll.
```
[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-member-roll-create.adoc]: "Service Mesh Member Roll" is the name of the object, so I've been capitalizing it.

**Comment by JStickler** [comment]
@bergerhoffer  I've updated based on your comments.  Thanks for the peer review.

**Comment by JStickler** [comment]
Requesting merge to 4.2 and 4.2 please.

**Comment by bergerhoffer** [comment]
@JStickler thanks for the updates! Merging.

**Pull request merged**